docs/samples/browser-plugin-meetings/app.js

```diff
@@ -409,25 +409,25 @@ function verifyPassword() {
     meeting
       .verifyPassword(joinOptions.pin, joinOptions.captcha)
       .then((res) => {
-        if (res.failureReason === 'NONE') {
+        if (res.isPasswordValid) {
           passwordCaptchaStatusElm.innerText = 'Password is verified';
           passwordCaptchaStatusElm.style.backgroundColor = '#49e849';
           verifyPasswordElm.disabled = true;
           document.getElementById('btn-join').disabled = false;
           document.getElementById('btn-join-media').disabled = false;
-          // joinMeetingNow();
-        } else if (res.failureReason === "WRONG_PASSWORD") {
-          passwordCaptchaStatusElm.innerText = 'Password is required';
-          passwordCaptchaStatusElm.style.backgroundColor = '#fa6e6e';
-        } else if (res.failureReason === 'WRONG_CAPTCHA') {
+        } else if (!res.isPasswordValid && res.requiredCaptcha) {
           passwordCaptchaStatusElm.innerText = 'Password & Captcha is required';
           passwordCaptchaStatusElm.style.backgroundColor = '#fa6e6e';
           meetingsJoinCaptchaImgElm.src = res.requiredCaptcha.verificationImageURL;
           meetingsJoinCaptchaImgElm.hidden = false;
           meetingsJoinCaptchaElm.type = "text";
           refreshCaptchaElm.hidden = false;
+        } else if (!res.isPasswordValid) {
+          passwordCaptchaStatusElm.innerText = 'Password is required';
+          passwordCaptchaStatusElm.style.backgroundColor = '#fa6e6e';
         } else {
           passwordCaptchaStatusElm.innerText = 'Check the logs for more info';
+          // might be possible if only captcha is required (without password) not handling here in samples
```

Comment:
hmm... I think this comment is not correct, because if only captcha is required then we will fall into the first condition check: `if (res.isPasswordValid)` from line 412 - maybe we should add `&& (!res.requiredCaptcha)` that condition in line 412. 
Also, looking into it more closely, maybe we have a bug in the SDK code itself: for the case if wbxappapi requires only captcha and no password, because then the client will need to call `meeting.verifyPassword()` to verify the captcha and probably passwordStatus will be NOT_REQUIRED and the code [here](https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/%40webex/plugin-meetings/src/meeting/index.js#L1046) doesn't account for that and will set `isPasswordValid` to false, which is probably incorrect

Reply:
I will remove this commented line. As you said, we have a bug in the SDK code itself. We need to handle only captcha and no password, Also While testing I didn't get this scenario anytime yet.